docs/mp/guides/10_mp-tutorial.adoc

```diff
@@ -545,7 +544,7 @@ Rebuild the application and run it.  After making some requests, update
 TIP: To learn more about Helidon MP configuration please see the 
  <<mp/config/01_introduction.adoc,Config>> section of the documentation.
 
-== Extending the application
+== Extending the Application
```

Comment:
Should this be "Extend" to be consistent with the earlier "Start..." and "Build..." headings?